api/src/main/kotlin/com/ford/internalprojects/peoplemover/tag/role/RoleRequest.kt

```diff
@@ -15,15 +15,15 @@
  * limitations under the License.
  */
 
-package com.ford.internalprojects.peoplemover.location
+package com.ford.internalprojects.peoplemover.tag.role
```

Comment:
This seems odd. git treated this as a file rename, rather than as a new file. I don't think this is an issue, though. 

This isn't an issue. git did this for us because it looks like LocationRequest, but with the Color attribute added and the file renamed. Technically, this is a whole new file. End result is the same. I'm just leaving this in as a note, but no action needs to be taken on it as far as the code goes.